server/lib/util.py

```diff
@@ -650,8 +650,11 @@ def post_body_cache_key():
   Builds flask cache key for POST requests using the request path and
   JSON-encoded post body
```

Comment:
please update comment since this is now used for GET too.

Reply:
Done!